frontend/src/App.vue

```diff
@@ -1,9 +1,16 @@
 <template>
   <div class="app">
     <nav-bar />
-    <character-bar v-if="!featureFlagStakeOnly && currentCharacterId !== null" />
     <div class="content dark-bg-text">
-      <router-view v-if="canShowApp" />
+      <b-row>
+        <b-col cols="3" lg="3" md="3" sm="2" :class="toggleSideBar ? 'sideBar triggerMargin' : 'sideBar removeMargin'">
+          <character-bar v-if="!featureFlagStakeOnly && currentCharacterId !== null" />
+        </b-col>
+        <b-col lg="9" md="9" sm="10" :class="toggleSideBar ? 'can-show-app addFlex  bg-image' : 'can-show-app removeFlex  bg-image'">
+          <router-view v-if="canShowApp" />
+        </b-col>
+        <WeaponRowGrid v-if="showWeapon" v-model="currentWeaponId" :checkForDurability="true"/>
+      </b-row>
```

Comment:
Some complicated css switching here can we make this a bit more simple. The Character-bar is outside the main app and is missing some checks. can we move it and some of the styling out of the app.vue

Reply:
> Some complicated css switching here can we make this a bit more simple. The Character-bar is outside the main app and is missing some checks. can we move it and some of the styling out of the app.vue

Im not sure, what happened there, In my end here's my code.
![code](https://user-images.githubusercontent.com/88535067/158749150-46992b95-5b1d-4464-9923-26d94789eed6.JPG)

Reply:
> Some complicated css switching here can we make this a bit more simple. The Character-bar is outside the main app and is missing some checks. can we move it and some of the styling out of the app.vue

I will only remove/move the css styling that I added.